### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by zmicer-rasko** [Multyjog/multyjog.github.io#1]
1. Move all pics into `/img` folder

**Comment by Multyjog** [comment]
This is a Task 4 me?

вс, 29 нояб. 2020 г., 03:38 zmicer-rasko <notifications@github.com>:

**Comment by zmicer-rasko** [comment]
@Multyjog no, you decided you will use another repo. There is a task for you https://github.com/Multyjog/sk.github.io/issues/1
Now rest young Jedi. And may the 4th be with you!
